src/python/aqueduct_executor/operators/connectors/tabular/relational.py

```diff
@@ -34,13 +42,18 @@ def delete(self, tables: List[str]) -> List[SavedObjectDelete]:
         metadata = MetaData()
         metadata.reflect(bind=self.engine)
         for table in tables:
+            exec_state = ExecutionState(user_logs=Logs())
             try:
                 sql_table = metadata.tables[table]
                 Base.metadata.drop_all(self.engine, [sql_table], checkfirst=True)
-            except:
-                results.append(SavedObjectDelete(name=table, succeeded=False))
+            except Exception as e:
```

Comment:
We can do this as a follow-up, consider ways to de-duplicate these code esp. for the exception handling part